### PR TITLE
Insert githubAccount as option

### PR DESCRIPTION
**Pull request by rhettl**

### CODE REVIEW

**Comment by rhettl** [comment]
And add to readme

File: generators/app/index.js [hunk 1]
```diff
@@ -64,6 +64,12 @@ module.exports = generators.Base.extend({
       desc: 'Project name'
     });
 
+    this.option('githubAccount', {
```
Comment (SBoudrias) [inline generators/app/index.js]: That's not used anywhere now.

Comment (rhettl) [inline generators/app/index.js]: It's used in `node:readme`, but `node:app` asks for `githubAccount` whether or not it's provided

File: generators/app/index.js [hunk 2]
```diff
@@ -188,7 +194,8 @@ module.exports = generators.Base.extend({
         this.prompt({
           name: 'githubAccount',
           message: 'GitHub username or organization',
-          default: username
+          default: username,
+          when: !this.options.githubAccount
```
Comment (SBoudrias) [inline generators/app/index.js]: We should actually skip the whole `githubUsername` block is the `githubAccount` option is provided. No need to make a server request if we'll skip it.

File: generators/app/index.js (hunk 1 above)
Comment (SBoudrias) [inline generators/app/index.js]: It's passing `this.props.githubAccount`, not `this.options.githubAccount`.

File: generators/app/index.js (hunk 2 above)
Comment (rhettl) [inline generators/app/index.js]: Good point, Altering

File: generators/app/index.js [hunk 3]
```diff
@@ -179,21 +185,25 @@ module.exports = generators.Base.extend({
...
-          username = username || '';
-        }
-        this.prompt({
-          name: 'githubAccount',
-          message: 'GitHub username or organization',
-          default: username
-        }, function (prompt) {
-          this.props.githubAccount = prompt.githubAccount;
-          done();
+      if (this.options.githubAccount) {
```
Comment (rhettl) [inline generators/app/index.js]: I rearranged this to prevent the system from making the server call. This also removes the need for `this.async`

**Comment by SBoudrias** [comment]
Allright, looks good! Squash your commits and we'll merge!

**Comment by rhettl** [comment]
Copy

**Comment by SBoudrias** [comment]
Awesome thanks!!

**Comment by rhettl** [comment]
:+1:

**Pull request merged**